marklogic-client-api/src/main/java/com/marklogic/client/datamovement/impl/WriteBatcherImpl.java

```diff
@@ -222,6 +225,7 @@
   private JobTicket jobTicket;
   private Calendar jobStartTime;
   private Calendar jobEndTime;
+  private DocumentMetadataHandle docMetadataHandle;

```

Comment:
Instead of using the type name for the field, could we use a name that suggests the use of the field?  Perhaps defaultMetadata?

That way, it's easier to understand for someone editing the code later (including oneself).